assets/js/components/ImageRadio.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * ImageRadio component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { MDCFormField, MDCRadio } from '../material-components';
+
+const ImageRadio = ( {
+	onClick,
+	id,
+	name,
+	value,
+	checked,
+	tabIndex,
+	onKeyDown,
+	image,
+	label,
+	children,
+	description,
+	ariaLabel,
+} ) => {
+	const formFieldRef = useCallback( ( el ) => {
+		if ( el !== null ) {
+			const formField = new MDCFormField( el );
+			const radioEl = el.querySelector( '.mdc-image-radio' );
+
+			if ( radioEl ) {
+				formField.input = new MDCRadio( radioEl );
+			}
+		}
+	}, [] );
+
+	return (
+		<div
+			className="mdc-form-field googlesitekit-image-radio"
+			ref={ formFieldRef }
+		>
+			<div className={ classnames( 'mdc-image-radio mdc-radio' ) }>
```

Comment:
I'm thinking this might've been copied from the `Radio` component, but if we're using `classnames`, let's separate each class:

```suggestion
			<div className={ classnames( 'mdc-image-radio', 'mdc-radio' ) }>
```

Or just use a plain string:

```suggestion
			<div className="mdc-image-radio mdc-radio">
```

(I prefer the first, but I'm cool with either; I think we use both in the codebase. But calling `classnames` for a single, static string seems off 😅 )